db_release_scripts/README.MD

```diff
@@ -1,144 +1,145 @@
-## MOSIP Commons module Databases (**mosip_master, mosip_kernel, mosip_audit...etc**) Release scripts inventory and deployment/revoke guidelines on postgresql database. 
+# MOSIP Commons Module Database Release Guide. 
 
 ## Prerequisities
 
-* DB Server and access details
+* DB Server and access details.
 
-* Postgres client (psql) has to be installed on the deployment servers.
+* Postgres client (psql) should be installed on the deployment servers.
 
-* Copy database release scripts(DDL, DML, .SH ... etc) from git/repository on to the DB deployment server.
+## Process
 
-* Necessary details needs to be updated in peoperties file against to the releavnt variables being used (details are listed below).
+### Release Script Transfer To Deployment Server
 
-* Database release objects related to MOSIP modules are placed in "**mosip_base_directory**>>db_release_scripts>>mosip_<schema_name> folder on git/repository
+*  Pull the DB deployment scripts to the deployment server directly and start deploying.
 
-**Example:** For commons module release script folder is /**mosip_base_directory**>>db_release_scripts>>mosip_master where all the database release scripts related to Master/Admin are available.
+*  For pulling in local system and pushing them back to deployment server using WinSCP then make a note to modify the following encoding settings in WinSCP before pushing the files to deployment server --> Open WinSCP --> Options --> Preferences --> Transfer --> Edit --> In "Transfer mode" section --> select "Text" --> Click Ok --> Click Ok.
```

Comment:
Add code syntax for winscp and the path.

Reply:
This is not a code or command snippet.